product_pricelist_alternative/models/product_pricelist_item.py

```diff
@@ -0,0 +1,32 @@
+# Copyright 2024 Camptocamp (<https://www.camptocamp.com>).
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import fields, models
+
+
+class PricelistItem(models.Model):
+    _inherit = "product.pricelist.item"
+
+    consider_alternative = fields.Selection(
+        selection=[
+            ("use_lower_price", "Use lower price"),
+            ("do_not_consider", "Do not consider"),
+        ],
+        default="use_lower_price",
+        required=True,
+    )
+
+    def _compute_price(self, product, quantity, uom, date, currency):
+        price = super(PricelistItem, self)._compute_price(
+            product, quantity, uom, date, currency
+        )
+        if self.consider_alternative == "use_lower_price":
```

Comment:
```suggestion
        if self.alternative_pricelist_policy == "ignore":
            return price
```
and un-indent the rest